JamJar/ChooseVideosViewController.swift

```diff
@@ -0,0 +1,281 @@
+//
+//  VideoStagingArea.swift
+//  JamJar
+//
+//  Created by Mark Koh on 5/11/16.
+//  Copyright © 2016 JamJar. All rights reserved.
+//
+
+import UIKit
+import DKImagePickerController
+import SCLAlertView
+import AVFoundation
+import Photos
+import KTCenterFlowLayout
+
+class VideoItem: UICollectionViewCell {
```

Comment:
Actually, can you make this into a separate class? I'd like to use this for the video collection view for JamJars.